modules/contact/src/actions/ContactAction.C

```diff
@@ -95,9 +95,9 @@ ContactAction::validParams()
   params.addParam<Real>("al_frictional_force_tolerance",
                         "The tolerance of the frictional force for augmented Lagrangian method.");
   params.addParam<Real>(
-      "c_normal", 1, "Parameter for balancing the size of the gap and contact pressure");
+      "c_normal", 1e6, "Parameter for balancing the size of the gap and contact pressure");
```

Comment:
Any helpful comments to guid the user's choice of that parameter? I'm thinking this is probably unit related?

Reply:
Added more explanation. But, for now, we may need to tune it for each type of problem.

Reply:
Yes, it is unit related. It has to do with the updates to the contact constraints.